.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: "\U0001F41BBug report"
+description: Create a report to help us improve Optuna.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please write a clear and concise description of what the bug is.
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: Please write a clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: |
+        Please give us some environment information. You can get this information by typing the following.
```

Comment:
```suggestion
        Please give us some environment information. You can get this information by executing the following commands on the terminal.
```

This passage was introduced by me (https://github.com/optuna/optuna/pull/2805) but I read it again and I'm now wondering if the suggestion would be more user-friendly. If you agree with me, please apply (or if you don't agree, just ignore).